main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/logger"
+	"github.com/kevinpita/go-todo/database"
+	"github.com/kevinpita/go-todo/routes"
+	"log"
+)
+
+func main() {
+	const port = 3000
+
+	app := fiber.New()
+	setupApp(app)
+
+	err := app.Listen(fmt.Sprintf(":%d", port))
+	if err != nil {
+		log.Fatalf("App could not be started on port %d\n", port)
+	}
+}
+
+func setupApp(app *fiber.App) {
+	app.Use(setupLogger())
+	db := database.CreateDatabase()
+	routes.RegisterRoutes(app, db)
```

Comment:
Same here

Reply:
Function `setupApp` has been changed to get the whole app setup [ea354a](https://github.com/kevinpita/go-todo/pull/1/commits/9ea354a04c041c3d7a90af4093b8004e36691484)